PhysicsTools/PatAlgos/python/slimming/miniAOD_tools.py

```diff
@@ -227,8 +227,15 @@ def miniAOD_customizeCommon(process):
     photon_ids = ['RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_25ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring15_50ns_V1_cff',
                   'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_25ns_nonTrig_V2p1_cff',
-                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff']
-    switchOnVIDPhotonIdProducer(process,DataFormat.MiniAOD, task)
+                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring15_50ns_nonTrig_V2p1_cff',
+                  'RecoEgamma.PhotonIdentification.Identification.cutBasedPhotonID_Spring16_V2p2_cff',
+                  'RecoEgamma.PhotonIdentification.Identification.mvaPhotonID_Spring16_nonTrig_V1_cff']
+    switchOnVIDPhotonIdProducer(process,DataFormat.AOD, task) 
```

Comment:
is this switch to DataFormat.AOD required?
Please clarify why.
Other IDs are done with .MiniAOD.

It apparently leads to some CPU increase (about 2.5% of the total miniAOD processing time).
If it's possible to avoid it without cost to physics, better use the other option.